### PR TITLE
correct full path to ALADIN config file & patch it to use right Fortran compiler flags

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyblocks/a/aladin.py [hunk 1]
```diff
@@ -144,16 +145,16 @@ def configure_step(self):
 
         # generate gmkpack configuration file
         self.conf_file = 'ALADIN_%s' % self.version
-        self.conf_filepath = os.path.join(self.builddir, 'arch', '%s.x' % self.conf_file)
+        self.conf_filepath = os.path.join(self.builddir, 'gmkpack_support', 'arch', '%s.x' % self.conf_file)
```
Comment (wpoely86) [inline easybuild/easyblocks/a/aladin.py]: no version if's?

File: easybuild/easyblocks/a/aladin.py [hunk 2]
```diff
@@ -144,16 +145,16 @@ def configure_step(self):
...
 
         try:
             if os.path.exists(self.conf_filepath):
                 os.remove(self.conf_filepath)
                 self.log.info("Removed existing gmpack config file %s" % self.conf_filepath)
 
-            archdir = os.path.join(self.builddir, 'arch')
+            archdir = os.path.dirname(self.conf_file)
```
Comment (wpoely86) [inline easybuild/easyblocks/a/aladin.py]: `self.conf_file = 'ALADIN_%s' % self.version` ? It's not a path?

File: easybuild/easyblocks/a/aladin.py (hunk 1 above)
Comment (boegel) [inline easybuild/easyblocks/a/aladin.py]: no, this was wrong all along...

File: easybuild/easyblocks/a/aladin.py (hunk 2 above)
Comment (boegel) [inline easybuild/easyblocks/a/aladin.py]: yeah, should be `self.conf_filepath`, fixing that

**Comment by boegel** [comment]
tested with https://github.com/hpcugent/easybuild-easyconfigs/pull/3292, works like a charm, going in

Thanks for the review @wpoely86!

**Pull request merged**